CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 1.7.21
+BREAKING CHANGES:
+* Exported bar file now does not use a "personium-localunit:" URL ([#512](https://github.com/personium/personium-core/issues/512))
```

Comment:
I think exported bar file uses
- In 1.7.17 before: `https://{cellName}.{domain}/__role/{boxName}`
- In 1.7.18-20: `personium-localunit:{cellName}:/__role/{boxName}`
- In 1.7.21 later: `https://{cellName}.{domain}/__role/__/`

Is this true? Better to describe above changes clearly.

 And better to add descriptions for bar file importing:
- OK: 1.7.17 => 1.7.17
- NG: 1.7.17 => 1.7.20
- NG: 1.7.20 => 1.7.20
- OK: 1.7.17 => 1.7.21
- NG: 1.7.20 => 1.7.21
- OK: 1.7.21 => 1.7.21

Reply:
Yes, it is true. I will describe the changes clearly.
For the compatibility, It will probably be difficult to fully describe the bar import behavior.
For example, if ACL is not defined in the bar file, then probably import will work.

Reply:
Ah, now I understood that bar file export is just broken after 1.7.18. after looking at the code below.
https://github.com/personium/personium-core/blob/develop/src/main/java/io/personium/core/bar/BarFileUtils.java#L106

You are right. bar file exported from 1.7.18-20 always fail. 
And also, bar file exported from both 1.7.17 and 1.7.21 will be imporable to both 1.7.17 and 21.
It is because the importer does not fully check the xml:base part.